tests/online_migration/sle12_online_migration/zypper_patch.pm

```diff
@@ -0,0 +1,31 @@
+# SLE12 online migration tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    my $self = shift;
+    select_console 'root-console';
+
+    # fully update the system
+    my $ret = zypper_call('patch --with-interactive -l');
+    die "zypper failed with code $ret" unless grep { $_ == $ret } (0, 102, 103);
+    $ret = zypper_call('patch --with-interactive -l', 2500);
+    die "zypper failed with code $ret" unless grep { $_ == $ret } (0, 102);
```

Comment:
this is third place where is this snippsed used .. maybe move to utils.pm ? as standard function?